### DISCUSSION TITLE
LunarVim in the Lazyman Neovim Configuration Manager

### DISCUSSION THREAD

**Discussion by doctorfree** [LunarVim/LunarVim#4153] [General]
I've created a Neovim Configuration Manager that supports easy installation and initialization of multiple Neovim configurations using the `NVIM_APPNAME` environment variable in Neovim 0.9.  I call it Lazyman and include support for `LunarVim` along with the fairly minimal custom LunarVim config at https://github.com/IfCodingWereNatural/minimal-nvim

See https://github.com/doctorfree/nvim-lazyman for info on Lazyman.

I'm contacting the maintainers of the Lazyman supported Neovim configurations to seek any feedback on the description, installation, and initialization of their config in Lazyman. Please feel free to let me know what I'm doing wrong or how I could improve the incorporation of `LunarVim` in Lazyman.

If you feel there is a need to modify the way `LunarVim` is used in Lazyman then open an issue at https://github.com/doctorfree/nvim-lazyman/issues

LunarVim is very cool. Thanks!

**Comment by LostInTheLogs** [comment]
LGTM, Cool project, feel free to ask/ping me if you have any issues/questions in the future
